pyxnat/tests/interfaces_test.py

```diff
@@ -54,3 +54,7 @@ def test_close_jsession():
 
 def test_save_config():
     central.save_config('/tmp/.xnat.cfg')
+
+def test_version():
+    v = central.version()
+    assert(v['tag'] == '1.7.5.1')
```

Comment:
Note that most stable XNAT releases have no `tag` item. Generally speaking I would better suggest using `version` instead since this might throw a `KeyError` exception in a future.

Reply:
Fixed in 3debe5d.